tests/application-registry-tests/test/k8stests/k8sresource_test.go

```diff
@@ -1135,4 +1200,92 @@ func TestK8sResources(t *testing.T) {
 
 	err = k8sResourcesClient.DeleteApplication(dummyApp.Name, &v1.DeleteOptions{})
 	require.NoError(t, err)
+
+	t.Run("when deleting application", func(t *testing.T) {
```

Comment:
I would suggest extracting this test case a separate test suite so that we maintain the Application per Test suite.

Reply:
Done!